src/utils/helper.js

```diff
@@ -23,8 +22,10 @@ function createDirectoryContents(
   nodeName,
   projectChoice,
   isThemeProvider,
-  isMaterialUI
+  isMaterialUI,
+  currentDirectory
 ) {
+  const CURR_DIR = currentDirectory ? currentDirectory : process.cwd();
```

Comment:
currentDirectory ? currentDirectory : process.cwd();
=>
currentDirectory || process.cwd();
if we can update, let's check in whole PR